R/map_admissions.R

```diff
@@ -25,9 +26,11 @@ map_admissions <- function(admissions, shapefile, date, scale_fill) {
     scale_fill <- scale_fill_viridis_c(option = "viridis", direction = -1)
   }
   
-  admissions <- admissions %>% 
-    filter(date %in% max_date)
-  
+  if (!is.null(max_date)) {
```

Comment:
Set above and it's to avoid the problem of filter(date %in% date) where dplyr will look within the data frame and error.

I was keeping it as a character string as I wasn't sure if `%in%` would work with dates (maybe it does?). Having that as the filter means you can pass a vector of dates to keep (if for example plotting cases every month). Not wedded to that though.

Reply:
Didn't realise part of the code (without changes) was hidden 🤦 

And not sure - I don't mind, it was just a very small annoyance

Reply:
I'll try it out locally and see what it does.

Reply:
done